modules/EvseV2G/v2g_server.cpp

```diff
@@ -163,7 +163,7 @@ static int v2g_incoming_v2gtp(struct v2g_connection* conn) {
         return -1;
     }
 
-    if (conn->payload_len >= UINT32_MAX - V2GTP_HEADER_LENGTH) {
+    if (conn->payload_len >= _UINT32_MAX - V2GTP_HEADER_LENGTH) {
```

Comment:
Good catch! We could also use UINT32_MAX from stdint instead of adding a extra define

Reply:
true, changed to the system one